### PR TITLE
Fixed small issues in identity group alias API docs

### PR DESCRIPTION
**Pull request by michelvocks**

### CODE REVIEW

File: website/source/api/secret/identity/group-alias.html.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ This endpoint creates or updates a group alias.
   corresponding existing group alias.
 
 - `mount_accessor` `(string: "")` – Mount accessor to which this alias belongs
-  toMount accessor to which this alias belongs to.
+  to.
```
Comment (ncabatoff) [inline website/source/api/secret/identity/group-alias.html.md]: Can you remove the first `to` in this sentence?  It reads awkwardly as-is.

File: website/source/api/secret/identity/group-alias.html.md [hunk 2]
```diff
@@ -74,7 +74,7 @@ This endpoint is used to update an existing group alias.
 - `name` `(string: entity-<UUID>)` – Name of the group alias.
 
 - `mount_accessor` `(string: "")` – Mount accessor to which this alias belongs
-  toMount accessor to which this alias belongs to.
+  to.
```
Comment (ncabatoff) [inline website/source/api/secret/identity/group-alias.html.md]: Ditto.

Comment (ncabatoff) [reply website/source/api/secret/identity/group-alias.html.md]: Alternatively, remove the second `to` in both sentences.  Either `to` is valid, just not both.

File: website/source/api/secret/identity/group-alias.html.md (hunk 1 above)
Comment (michelvocks) [reply website/source/api/secret/identity/group-alias.html.md]: Ty. Updated

File: website/source/api/secret/identity/group-alias.html.md (hunk 2 above)
Comment (michelvocks) [reply website/source/api/secret/identity/group-alias.html.md]: Ty. Updated

**Pull request merged**